connectors/katalog/pkg/connector/connector_test.go

```diff
@@ -95,3 +100,217 @@ func TestGetAssetInfo(t *testing.T) {
 		g.Expect(response.Credentials).To(BeEquivalentTo("/v1/kubernetes-secrets/creds-demo-asset?namespace=demo"))
 	})
 }
+
+func TestCreateAssetInfo(t *testing.T) {
+	t.Parallel()
+	g := NewGomegaWithT(t)
+	logf.SetLogger(zap.New(zap.UseDevMode(true)))
+
+	s3Connection := taxonomy.Connection{
+		Name: "s3",
+		AdditionalProperties: serde.Properties{
+			Items: map[string]interface{}{
+				"s3": map[string]interface{}{
+					"endpoint":   "s3.eu-gb.cloud-object-storage.appdomain.cloud",
+					"bucket":     "fybrik-test-bucket",
+					"object_key": "small.csv",
+				},
+			},
+		},
+	}
+	var csvFormat taxonomy.DataFormat = "csv"
+	var sourceAssetName = "paysim-csv"
+	var destAssetName = "new-paysim-csv"
+	var destCatalogID = "fybrik-system"
+
```

Comment:
styling: destCatalogID := "fybrik-system"
same for all vars

Reply:
Fixed. Please check. Thanks.